classes/author/PKPAuthor.inc.php

```diff
@@ -1,21 +1,21 @@
 <?php
 
 /**
- * @file classes/submission/PKPAuthor.inc.php
+ * @file classes/author/PKPAuthor.inc.php
```

Comment:
The file should be renamed `Author.inc.php` and the class named `PKP\author\Author`. The `PKP` prefix is no longer needed now that we're using namespaces.

Reply:
Changed it everywhere